2018/readme.md

```diff
@@ -0,0 +1,128 @@
+
+Contains methods that manipulate LAR or TS dataframes to fail a specific syntax or validity edit. Because the test_data class changes LAR rows sequentially and with set values, it may fail other edits while changing values in the synthetic LAR to fail a specific edit.
+
+Note: changes to the clean data do not create exhaustive test cases for each edit. 
+
+Each edit logic method in `lar_constraints`, `test_data`, and `rules_engine` is prefaced with 's', 'v', or 'q' to indicate a syntax, validity or quality edit respectively.
+
+## Configurations 
+The following yaml configuration files and values can be changed to alter the output of HMDA submission test files. 
+
+### [Test Files Configuration](https://github.com/cfpb/hmda-test-files/blob/master/2018/python/configurations/test_filepaths.yaml)
+
+| Key| Default Value | Expanation  |
+|:-------------|:-------------|:--------|
+|clean_filepath | "../edits_files/clean_files/{bank_name}/" | Filepath for generated clean files. Note: the filepath needs to contain `{bank_name}` for formatting the institution name to the path. |
+| clean_filename|"clean_file_{n}_rows_{bank_name}.txt"| Filename for generated clean files. Note: the filepath needs to contain `{bank_name}` and `{n}` for formatting the institution name and number of rows to the filename. |
+| validity_filepath | "../edits_files/test_files/{bank_name}/validity/"| Filepath for generated validity edit test files. Note: the filepath needs to contain `{bank_name}` for formatting the institution name to the path.|
+|syntax_filepath |"../edits_files/test_files/{bank_name}/syntax/"|Filepath for generated syntax edit test files. Note: the filepath needs to contain `{bank_name}` for formatting the institution name to the path. 
+|quality_filepath | "../edits_files/test_files/{bank_name}/quality/"|Filepath for generated quality edit test files. Note: the filepath needs to contain `{bank_name}` for formatting the institution name to the path. 
+| quality_pass_s_v_filepath|"../edits_files/test_files/{bank_name}/quality_pass_s_v/"| Filepath for generated quality edit test files that also pass syntax and validity edits. Note: the filepath needs to contain `{bank_name}` for formatting the institution name to the path |
+ts_schema_json | "../schemas/ts_schema.json"| Filename for the Transmital Sheet schema. 
+lar_schema_json| "../schemas/lar_schema.json"| Filename for the Loan Application Register schema. |
+edit_report_filepath| "../edits_files/"|Filepath for storing generated edit reports.|
+edit_report_filename| "edit_report.csv"|Filename for generated edit reports.|
+log_filename | "clean_files_log.txt" | Filename for the log file detailing iterations in the file creation process.|
+log_mode| 'a'| The mode for logging files. Currently, the log mode is set to append log entries ('a'). This configuration can be changed to write over the previous file ('w').
```

Comment:
Please change the default value to overwrite existing log files (both here and in the config file)

Reply:
Sure! Changes have been made.